src/InfluxDB/Point.php

```diff
@@ -81,26 +88,39 @@ public function __toString()
         return $string;
     }
 
+    /**
+     * @param  array $arr
+     * @return string
+     */
     private function arrayToString(array $arr)
     {
         $strParts = array();
 
         foreach ($arr as $key => $value) {
-            $strParts[] = "{$key}={$value}";
+            $strParts[] = sprintf('%s=%s', $key, $value);
```

Comment:
Hahahahaha. Well, isn't that bad a `sprintf`. You could pin it to the project code-style. It's better than have a mix of many flavours:

- `$key . '=' . $value`;
- `"{$key}={$value}"`;
- `sprintf('%s=%s', $key, $value)`.

Agree?